core/templates/pages/exploration-editor-page/services/user-email-preferences-backend-api.service.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { UrlInterpolationService } from 'domain/utilities/url-interpolation.service';
+import { ExplorationDataService } from './exploration-data.service';
+import { RequestParams } from './user-email-preferences.service';
+
+/**
+ * @fileoverview User exploration emails backend api service
+ * for the exploration settings.
+ */
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UserEmailPreferencesBackendApiService {
+  constructor(
+    private http: HttpClient,
+    private urlInterpolationService: UrlInterpolationService,
+    private explorationDataService: ExplorationDataService
+  ) { }
+
+  saveChangeToBackend(
+      requestParams: RequestParams
+  ): Promise<void | object> {
```

Comment:
Can you find out if this returns void or returns an Object. If it returns an Object, find the exact type. Object is too generic.

Reply:
Done.

Reply:
I found the exact type.
But lint tests fails when I use a type other an object.
Everything works as expected on the web app.
What should I do disable lint checks for this or use the type object.
![Screenshot from 2021-02-25 12-59-05](https://user-images.githubusercontent.com/24855641/109118481-61bd4f00-7769-11eb-9f0a-d700e79c7ec0.png)

Reply:
Done.